src/psyclone/psyir/nodes/omp_directives.py

```diff
@@ -2199,6 +2199,22 @@ def __init__(self, **kwargs):
         self.addchild(OMPDefaultClause(
             clause_type=OMPDefaultClause.DefaultClauseTypes.SHARED))
 
+    @staticmethod
+    def create(children=None, **kwargs):
+        '''
+        Create an OMPParallelDoDirective.
+
+        :param children: The child nodes of the new directive.
+        :type children: List of :py:class:`psyclone.psyir.nodes.Node`
+        :param kwargs: additional keyword arguments provided to the PSyIR node.
+        :type kwargs: unwrapped dict.
```

Comment:
Can you add documentation for the OMPDoDirective keyword arguments into the kwargs (omp_schedule, collapse and reprod).

Also could you help us and mark the omp_schedule argument in OMPDoDirective as optional as well.

Reply:
I added these using a list-like format as I'm not sure if you have a specification for listing `kwargs` in docstrings, let me know if this is fine or not.

Reply:
Apologies - please remove these after all.

Reply:
No problem.

Reply:
One other thing on docs I forgot to mention apologies. The `\` to signal a newline in the docs we no longer use - can you fix this here and in the OMPDoDirective docs that you fixed the indentation in? Then I'll submit this for integration tests.

Reply:
I took the opportunity to get rid of them in all docstrings in this file and also to indent all of them correctly. Apologies for the longer review.